pkg/karmadactl/karmadactl.go

```diff
@@ -90,6 +90,7 @@ func NewKarmadaCtlCommand(cmdUse, parentCommand string) *cobra.Command {
 				NewCmdLogs(f, parentCommand, ioStreams),
 				NewCmdExec(f, parentCommand, ioStreams),
 				NewCmdDescribe(f, parentCommand, ioStreams),
+				NewCmdInterpret(f, parentCommand, ioStreams),
```

Comment:
I guess this could be a advanced command, is it?

Reply:
This command is used to help users finding the mistakes in customizations. So it's a troubleshooting and debugging command.